docs/apis-tools/go-client/index.md

```diff
@@ -7,14 +7,14 @@ description: "Instantiate the client by passing in the address of the cluster yo
 
 ## Dependencies
 
-To use the Zeebe Go client library, add the following dependency to your `go.mod`:
+To use the [Zeebe Go client library](https://github.com/camunda/camunda-platform-get-started/tree/main/go), add the following dependency to your `go.mod`:
 
 ```
 module github.com/zb-user/zb-example
 
-go 1.17
+go 1.20
```

Comment:
Same as my other comment. ATM, we are [using go 1.19](https://github.com/camunda/zeebe/blob/1c667b26030d76b8679948a8f5e20e65ffefaa78/clients/go/go.mod#L3C2-L3C2). Since I'm not a go expert; not sure if this makes a big difference

Reply:
```suggestion
go 1.19
```